projects/plugins/jetpack/extensions/blocks/business-hours/style.scss

```diff
@@ -8,7 +8,7 @@
 
 	dt {
 		font-weight: bold;
-		margin-right: 0.5em;
+		margin-inline-end: 0.5em;
```

Comment:
Styles are still registered and enqueued manually in PHP. Eventually, they'll be determined from `block.json` and we won't be able to specify `view.css` or `view.rtl.css` at runtime. Let's write code that supports internationalization from the get-go.

Reply:
Good catch!